client/src/app/components/assign-stock-keepers/assign-stock-keepers.component.spec.ts

```diff
@@ -33,4 +33,69 @@ describe('AssignStockKeepersComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  // Test that Start Audit button is disabled when no items selected
+  it('should enable Assign button when item(s) selected', () => {
+    fixture.detectChanges();
+    const checkboxArray = fixture.debugElement.queryAll(By.css('.mat-checkbox-input'));
+    expect(checkboxArray.length).toBe(0);
+    for (const input  of checkboxArray){
+        expect(input.nativeElement.checked).toBeFalsy();
+        input.nativeElement.click();
+        fixture.detectChanges();
+        const button = fixture.debugElement.query(By.css('.button.bottom-assign'));
+        expect(button.nativeElement.disabled).toBeFalsy();
+      }
+  });
+
+  // Test the deleteAudit()
+  it('Call deleteAudit', () => {
+    try {
+      component.deleteAudit();
+      component.discardAudit();
+    }
+    catch (errorMessage) {
+      console.error(errorMessage);
+    }
+  });
+
+  // Test the goBackIventory()
+  // it('Call goBackIventory', () => {
+  //   try {
+  //     component.goBackIventory();
+  //   }
+  //   catch (err) {
+  //   console.error(err);
+  //   }
+  // });
```

Comment:
if this isn't needed, let's delete it

Reply:
@NicholasNagy in response to your comments, for the upcoming changes that will be made the reason these tests are commented out is to show others how to setup their tests / serve as an example

Reply:
okay, cool, can you just add a comment explaining that right above that so it doesn't accidentally get deleted?

Reply:
Yes makes sense, for clarifications. Just added the comments in the code.